invenio_app_rdm/users_ui/templates/semantic-ui/invenio_app_rdm/users/header.html

```diff
@@ -23,14 +23,14 @@ <h1 class="ui medium header" aria-label="{{ _('Your dashboard') }}">{{ current_u
       </div>
     </div>
     {% if active_dashboard_menu_item == "uploads" %}
-      <a class="ui tiny button positive left labeled icon m-0" href="/uploads/new">
+      <a class="ui tiny button positive left labeled icon m-0" href={{ config.APP_RDM_ROUTES['deposit_create'] }}>
```

Comment:
```suggestion
      <a class="ui tiny button positive left labeled icon m-0" href={{ url_for("deposit_create") }}>
```

minor/shelve: in theory it's the same, since the code paths are technically the same. I think we're missing though passing an explicit `endpoint` parameter when registering the routes, which means that they get auto-generated from the function name (which also caused the issue you mentioned). 

Fixing the explicit endpoint naming is a bit out of scope and a bigger task, so I'll shelve it, but starting to use `url_for` where possible would be good.

Reply:
Out of curiosity, what is the advantage of that? Normalization on how do we access the routes?

Reply:
Yeah, I would prefer using `url_for` and having explicit endpoint names, since this is a Flask thing vs. an Invenio(RDM) thing (i.e. less to learn for someone new to the codebase).